soccer/src/soccer/strategy/agent/position/offense.cpp

```diff
@@ -367,6 +374,14 @@ communication::PosAgentResponseWrapper Offense::receive_communication_request(
 
         comm_response.response = response;
         return comm_response;
+    } else if (const communication::SeekerRequest* seeker_request =
```

Comment:
```suggestion
if (const communication::SeekerRequest* seeker_request =
```
Unnecessary after reutrn